script-templates/python/lib/script_support/lab_control.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 Skytap Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests
+import json
+from skytap_metadata import SkytapMetadata
+
+class LabControl:
```

Comment:
I haven't used python so this syntax all looks a bit strange to me, but it seems like it should work...

Reply:
This was the first python I've ever written, and I'm also not a fan, but it seems to work!